src/components/Notification/Notification.stories.mdx

```diff
@@ -27,8 +27,7 @@ import DSProvider from "../../theme/provider";
   parameters={{
     design: {
       type: "figma",
-      url:
-        "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/?node-id=10734%3A812",
+      url: "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/?node-id=10734%3A812",
```

Comment:
Are these figma changes auto-generated?  Wondering if it's a bad merge somewhere - they look right but should I revert?

Reply:
It might be prettier making this change.

Reply:
The only difference was the line break - my vscode mdx lint settings are off and I'm not quite sure how to fix it.  If it isn't a huge deal I'd rather not deal with it right now either